cmake/scripts/windows/ArchSetup.cmake

```diff
@@ -45,12 +45,9 @@ set(SYSTEM_DEFINES -DWIN32_LEAN_AND_MEAN -DNOMINMAX -DHAS_DX -D__STDC_CONSTANT_M
                    -DPLT_HTTP_DEFAULT_USER_AGENT="UPnP/1.0 DLNADOC/1.50 Kodi"
                    -DPLT_HTTP_DEFAULT_SERVER="UPnP/1.0 DLNADOC/1.50 Kodi"
                    -DUNICODE -D_UNICODE
+                   -DFRIBIDI_STATIC
```

Comment:
this is the wrong place to add a define for a library

Reply:
Why? These lines are for libraries
```
-DTAGLIB_STATIC -DNPT_CONFIG_ENABLE_LOGGING
-DPLT_HTTP_DEFAULT_USER_AGENT="UPnP/1.0 DLNADOC/1.50 Kodi"
-DPLT_HTTP_DEFAULT_SERVER="UPnP/1.0 DLNADOC/1.50 Kodi"
```

Reply:
`TAGLIB_STATIC` is wrong too, because you tell in general that the library is built static on windows, but that's not true. A non static variant would work too, but then the define is wrong.
`PLT_HTTP_DEFAULT_USER_AGENT` and `PLT_HTTP_DEFAULT_SERVER` should be defined at a common place for all platforms, because they are currently only defined for Windows and WindowsStore.